uaclient/tests/test_update_messaging.py

```diff
@@ -605,18 +616,16 @@ def test_message_based_on_beta_status_and_count_until_active_esm(
             cfg.override_features({"allow_beta": cfg_allow_beta})
 
         m_entitlement_cls = mock.MagicMock()
-        type(m_entitlement_cls).is_beta = is_beta
+        m_entitlement_cls.is_beta = is_beta
```

Comment:
Is there a reason we want to change this from a class type declaration to a direct attribute set?
I **think** the right way to do this is with [mock.PropertyMock](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.PropertyMock).

I think that would result in:
```suggestion
        type(m_entitlement_cls).is_beta = mock.PropertyMock(return_value=is_beta)
```

Reply:
I see it was like this before.
From the mocking manipulation perspective, it will be the same.
From the semantics, having the property mocks makes more sense - the `type(` thing looked ugly to me.